pulsar-io/core/src/main/java/org/apache/pulsar/io/core/Message.java

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
How come this is showing up as new file?

Reply:
@srkukarni Probably because of the merge. I'll remove it.